src/Controller/Backend/BackendBase.php

```diff
@@ -42,6 +42,14 @@ protected function render($template, array $context = [], array $globals = [])
             $context = ['context' => $context];
         }
 
+        /** @deprecated Deprecated since 3.0, to be removed in 4.0. */
+        $oldGlobals = $this->app['twig']->getGlobals();
+        foreach (['user', 'users', 'bolt_version'] as $key) {
+            if (!isset($context['context'][$key])) {
+                $context['context'][$key] = $oldGlobals[$key];
+            }
+        }
```

Comment:
If we don't want them as globals, and we want to pass them for all backend routes (which I may have a problem with), then they should be pulled the proper way, not from Twig globals.